src/content/components/PocketNewtabView/PocketNewtabView.js

```diff
@@ -207,7 +207,7 @@ export class PocketNewtabView extends React.PureComponent {
     for (const bug of bugs) {
       if (bug.summary.toLowerCase().startsWith("[meta]")) {
         continue;
-      } else if (["P1", "P2"].includes(bug.priority)) {
+      } else if (bug.priority === "P1") {
```

Comment:
change here is pushing P2 bugs fixed earlier in backlog which doesn't look right

Reply:
For Pocket New Tab we should use both P1 and P2 as it shows consolidated view of multiple releases

Reply:
ok I'll leave it as is for now, let's discuss tomorrow